<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewer</class>
 <widget class="QMainWindow" name="viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="text_fields_layout">
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>next point x: </string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>delta</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="delta_text">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="x_text">
       <property name="text">
        <string>500</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="y_text">
       <property name="text">
        <string>300</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>next point y:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>251</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="colors_layout">
     <item row="1" column="2">
      <widget class="QRadioButton" name="white_pen_but">
       <property name="text">
        <string>white</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>pen color</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="green_pen_but">
       <property name="text">
        <string>green</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="yellow_pen_but">
       <property name="text">
        <string>yellow</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QRadioButton" name="magenta_pen_but">
       <property name="text">
        <string>magenta</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="red_pen_but">
       <property name="text">
        <string>red</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="cyan_pen_but">
       <property name="text">
        <string>cyan</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>246</width>
      <height>146</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QPushButton" name="set_seed_but">
       <property name="text">
        <string>set seed</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="erase_but">
       <property name="text">
        <string>erase</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="add_point_but">
       <property name="text">
        <string>add point</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="fill_but">
       <property name="text">
        <string>fill figure ( = Enter)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="add_hole_but">
       <property name="text">
        <string>add hole ( = Shift)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="add_vertical_but">
       <property name="text">
        <string>add vertical</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="add_horisontal_but">
       <property name="text">
        <string>add horisontal</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>251</width>
      <height>82</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QRadioButton" name="without_delay_but">
       <property name="text">
        <string>without delay</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="with_delay_but">
       <property name="text">
        <string>with delay</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>delay (in ms):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="delay_text">
       <property name="text">
        <string>15</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="colors_layout_2">
     <item row="2" column="1">
      <widget class="QRadioButton" name="cyan_brush_but">
       <property name="text">
        <string>cyan</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>brush color</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QRadioButton" name="white_brush_but">
       <property name="text">
        <string>white</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="yellow_brush_but">
       <property name="text">
        <string>yellow</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QRadioButton" name="magenta_brush_but">
       <property name="text">
        <string>magenta</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="green_brush_but">
       <property name="text">
        <string>green</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="red_brush_but">
       <property name="text">
        <string>red</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>70</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>241</width>
      <height>46</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QRadioButton" name="draw_mode_but">
       <property name="text">
        <string>draw</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="seed_mode_but">
       <property name="text">
        <string>seed</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>mode</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1278</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
